bridges/ironSource/app/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    defaultConfig {
+        //applicationId "com.loopme.ironsource_mediation_sample"
+        minSdkVersion 24
+        compileSdk 34
+        targetSdkVersion 34
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
+    }
+    namespace 'com.loopme.ironsource_mediation_sample'
+}
+
+repositories {
+    maven {
+        url 'https://android-sdk.is.com/'
+    }
+    mavenCentral()
+}
+
+dependencies {
+
+    implementation 'androidx.appcompat:appcompat:1.6.1'
+    implementation 'com.google.android.material:material:1.10.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'com.ironsource.sdk:mediationsdk:7.7.0'
+    implementation project(path: ':loopme-sdk')
```

Comment:
L43-44 should be removed, because adapter already has IronSourceSDK and LoopMeUnitedSDK dependency